### PR TITLE
Add `owner` flag to generate an optional `owner` token.

### PR DESCRIPTION
**Pull request by leerosen-cape**

### CODE REVIEW

File: cmd/cape/cmd/token.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ func init() {
 	rootCmd.AddCommand(tokenCmd)
 
 	tokenCmd.PersistentFlags().IntP("expires", "e", 3600, "optional time to live (in seconds)")
+	tokenCmd.PersistentFlags().BoolP("owner", "", false, "optional owner token")
```
Comment (gavinuhma) [inline cmd/cape/cmd/token.go:33]: Thinking the message could be more descriptive

**Review by gavinuhma — DISMISSED** [review]

**Comment by leerosen-cape** [comment]
@gavinuhma @bendecoste Can I get another :heavy_check_mark: on this? Will expand the flag description with uses as they come, any other thoughts on that?

File: cmd/cape/cmd/token.go [hunk 2]
```diff
@@ -52,14 +58,20 @@ func token(cmd *cobra.Command, args []string) error {
...
 	}
 
+	var audience []string
+	if owner {
+		audience = append(audience, "owner")
+	}
+
 	token := jwt.NewWithClaims(jwt.SigningMethodRS256, jwt.RegisteredClaims{
 		Subject:   functionID,
+		Audience:  audience,
```
Comment (bendecoste) [inline cmd/cape/cmd/token.go:74]: Why do we use the audience field for this?

Comment (leerosen-cape) [reply cmd/cape/cmd/token.go:74]: The token "audience" in this case, is the "owner" themselves.

Comment (bendecoste) [reply cmd/cape/cmd/token.go:74]: I believe the audience claim is generally used to identify who the token is meant to be processed by? This seems like a token that has elevated privileges, so a custom claim might be in order?

Comment (leerosen-cape) [reply cmd/cape/cmd/token.go:74]: @bendecoste AFAIK the `aud` or audience is regarded as the `recipients` the token is intended for (a string array)

Comment (leerosen-cape) [reply cmd/cape/cmd/token.go:74]: Being said, given its use in oAuth I will add a custom claim

**Review by gavinuhma — APPROVED** [review]

File: cmd/cape/cmd/token.go (hunk 2 above)
Comment (bendecoste) [reply cmd/cape/cmd/token.go:74]: > @bendecoste AFAIK the `aud` or audience is regarded as the `recipients` the token is intended for (a string array)

🤔 For example the token that auth0 issues to a user has the `aud` set as `something.capeprivacy.com` (login right now I think). Which is the server that processes the token, although the token is given to the user.

**Comment by leerosen-cape** [comment]
@bendecoste Updated to use `scope` claim.

Non-owner token example:
```
{
  "exp": 1662132232,
  "iat": 1662128632,
  "scope": "function:invoke",
  "sub": "o42RHwadCWBJatQbHF9Eto"
}
```

Owner token:
```
{
  "exp": 1662132285,
  "iat": 1662128685,
  "scope": "function:invoke function:output",
  "sub": "o42RHwadCWBJatQbHF9Eto"
}
```

**Review by bendecoste — APPROVED** [review]

**Pull request merged**